site/styles/_meta-detail.scss

```diff
@@ -590,3 +576,15 @@
     font-size: 12px;
   }
 }
+
+s72-element-switcher {
+  @extend .mb-5;
+}
+
+.meta-detail-infos {
```

Comment:
Nit: I think should be singular. It doesn't matter the class is used multiple times.

```suggestion
.meta-detail-info {
```

Reply:
- changed to singular